infrastructure/templates/template.json

```diff
@@ -2361,6 +2365,7 @@
         "ResourceGroupName": "[resourceGroup().name]",
         "SubscriptionId": "[subscription().subscriptionId]",
         "TenantId": "[subscription().tenantId]",
+        "WEBSITE_TIME_ZONE ": "[parameters('timeZone')]",
```

Comment:
This might need to be in local.settings.json as well to test effectively locally.